pyproject.toml

```diff
@@ -110,6 +110,7 @@ ignore = [
   "EM101",    # Exceptions should not use string literals
   "SIM108",   # Use ternary operator. Sometimes this is less readable.
   "T201",     # Remove 'print' statements. Sometimes these are handy.
+  "SIM105",   # Use contextlib.suppress instead of try-except-pass. The recommended approach is slower and less flexible.
```

Comment:
I dislike this rule too. We have an open PR on icepyx where we're discussing this! https://github.com/icesat2py/icepyx/pull/572